Packs/PAN-OS/Integrations/Panorama/README.md

```diff
@@ -3624,10 +3626,10 @@ Installs the target PAN-OS version on the specified target device.
 `pan-os-install-panos-version`
 #### Input
 
-| **Argument Name** | **Description** | **Required** |
-| --- | --- | --- |
-| target | The target device on which to install the target PAN-OS software version. | Optional | 
-| target_version | Target PAN-OS version to install. | Required | 
+| **Argument Name** | **Description**                                                                              | **Required** |
+| --- |----------------------------------------------------------------------------------------------| --- |
```

Comment:
Why?

Reply:
fixed it i guess it happens automaticlly with generate docs